src/cljs_test_runner/main.clj

```diff
@@ -44,21 +44,24 @@
         src-path (str/join "/" [src "runner.cljs"])
         out-path (str/join "/" [out "test-runner.js"])
         {:keys [target doo-env]} (case env
-                                   :node {:target :nodejs
-                                          :doo-env :node}
-                                   :phantom {:target :browser
+                                   :node    {:target  :nodejs
+                                             :doo-env :node}
+                                   :phantom {:target  :browser
                                              :doo-env :phantom})]
```

Comment:
Just a style thing, but I don't tend to align things like this. If you're following a style guide or something that has some good cases for this I'll happily consider it. Would rather there wasn't any in line padding to maintain though, normally.

(thinking about it, maybe it's something your editor did automatically?)